Source/WebCore/workers/service/ServiceWorkerGlobalScope.cpp

```diff
@@ -140,6 +142,8 @@ Page* ServiceWorkerGlobalScope::serviceWorkerPage()
 
 void ServiceWorkerGlobalScope::skipWaiting(Ref<DeferredPromise>&& promise)
 {
+    RELEASE_LOG(ServiceWorker, "ServiceWorkerThread::skipWaiting for worker %" PRIu64, thread().identifier().toUInt64());
```

Comment:
This should be `ServiceWorkerGlobalScope::skipWaiting`?

Reply:
ok